.github/workflows/tests.yaml

```diff
@@ -0,0 +1,41 @@
+name: Tests
+
+on:
+  push:
+    branches: [ master, v5 ]
+  pull_request:
+    branches: [ master, v5 ]
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  ansible:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Install Ansible
+        run: |
+          python -m pip install --upgrade wheel
+          python -m pip install --upgrade ansible
+
+      - name: Run the all-in-one playbook
+        run: |
+          if test -f ansible/all-in-one/site.yml; then
+            # v5
+            ansible-playbook --extra-vars '{"require_cgroupsv2": false}' -vv --inventory "xsnippet," --connection=local ansible/all-in-one/site.yml
```

Comment:
Q: Why `require_cgroupsv2` are disabled? Because of no intention to use rootless podman?

Reply:
I'm not sure if you've seen #33 yet, but configuring cgroupsv2 requires a machine reboot, so I don't think that's going to fly in the GH Workflows virtual environment where we run the Ansible playbook on the same (virtual) host that we configure.

To be honest, I am puzzled by the requirement of cgroupsv2 by rootless Podman: the docs say it _is_ required, but it seems to be working just fine w/o it. The only thing that did not work for me is displaying pod stats (CPU/memory usage, I/O operations, etc).

So, basically, I want to have cgroupsv2 configured in prod, but want to skip it when the playbook is executed in the GH virtual environment.